uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/AbsenceType.php

```diff
@@ -471,4 +471,25 @@ public static function getEnabledAbsenceTypes()
 
     return $absenceTypes;
   }
+
+  /**
+   * Returns the AbsenceType where the must_take_public_holiday_as_leave option
+   * is set
+   *
+   * @return \CRM_HRLeaveAndAbsences_BAO_AbsenceType|null
+   */
+  public static function getOneWithMustTakePublicHolidayAsLeaveRequest() {
```

Comment:
the name is a bit odd to me .. maybe 
getPublicHolidayLeaveType

or maybe make it a generic one that return any LeaveType based on based passed parameters , not sure which is best here .